pkg/testutil/helpers.go

```diff
@@ -287,3 +288,46 @@ func componentCheck(
 	}
 	t.Logf(fmt.Sprintf("%s is ready", componentName))
 }
+
+func NewProviderAccount(name string, subjects ...rbacv1.Subject) *catalogv1alpha1.Account {
```

Comment:
This should be in a separate PR in the future since refactoring `framework.go` is orthogonal to the account service. You could make stacked PR, and we'd quickly approve the refactoring PR.

Reply:
This should be in this PR, because the account integration test(which doesn't have a framwork pointer) needs to use it, that's why I remove the receiver.